api/v1alpha1/pool_types.go

```diff
@@ -45,10 +45,9 @@ type PoolSpec struct {
 
 // PoolStatus defines the observed state of Pool
 type PoolStatus struct {
-	ID          string `json:"id"`
-	IdleRunners uint   `json:"idleRunners"`
-	Runners     uint   `json:"runners"`
-	Selector    string `json:"selector"`
+	ID                     string `json:"id"`
+	LongRunningIdleRunners uint   `json:"longRunningIdleRunners"`
```

Comment:
@rafalgalaw wdyt about renaming the `poolStatus` struct here?
We should definetly get rid of `Runners` in here as it's not super helpful and during business hour it will force a reconciliation each time we try to reflect the current number of runners in here.

I've renamed `IdleRunners` to `LongRunningIdleRunners` to make it more clear, that this field will only reflect the number of ilde runners older than ? time.Period

Reply:
I am fine with this, as this expresses our intention on how we even scale those pools down 👍